projects/vdk-plugins/vdk-test-utils/src/vdk/plugin/test_utils/util_funcs.py

```diff
@@ -135,6 +135,14 @@ def jobs_path_from_caller_directory(job_name: str):
     return get_test_job_path(caller_dir, job_name)
 
 
+def get_caller_directory():
+    import inspect
+
+    frame = inspect.stack()[1]
+    module = inspect.getmodule(frame[0])
+    return pathlib.Path(os.path.dirname(os.path.abspath(module.__file__)))
```

Comment:
this line could be reused(composed) in favour of copy-paste from another part of util_funcs.py

Reply:
The problem with reusing this in, say, jobs_path_from_caller_directory is that the stack trace would change. I will see if I can make it work both when called directly and indirectly, via another function.

Reply:
The problem with reusing this in, say, `jobs_path_from_caller_directory` is that the stack trace would change. I will see if I can make it work both when called directly and indirectly, via another function.